doc/src/modules/physics/continuum_mechanics/beam_problems.rst

```diff
@@ -209,19 +209,19 @@ with:
    :include-source: True
 
    >>> b.slope()
-     ⎛                            3                          3             ⎞ 
-     ⎜         1         2   4⋅<x>              1   4⋅<x - 5>             2⎟ 
-    -⎜- 158⋅<x>  + 26⋅<x>  - ────── - 50⋅<x - 5>  + ────────── - 6⋅<x - 9> ⎟ 
-     ⎝                         3                        3                  ⎠ 
+     ⎛                            3                          3             ⎞
+     ⎜         1         2   4⋅<x>              1   4⋅<x - 5>             2⎟
+    -⎜- 158⋅<x>  + 26⋅<x>  - ────── - 50⋅<x - 5>  + ────────── - 6⋅<x - 9> ⎟
+     ⎝                         3                        3                  ⎠
     ─────────────────────────────────────────────────────────────────────────
-                                       E⋅I                                   
+                                       E⋅I
    >>> b.deflection()
-     ⎛                  3      4                        4             ⎞ 
-     ⎜        2   26⋅<x>    <x>              2   <x - 5>             3⎟ 
-    -⎜- 79⋅<x>  + ─────── - ──── - 25⋅<x - 5>  + ──────── - 2⋅<x - 9> ⎟ 
-     ⎝               3       3                      3                 ⎠ 
+     ⎛                  3      4                        4             ⎞
+     ⎜        2   26⋅<x>    <x>              2   <x - 5>             3⎟
+    -⎜- 79⋅<x>  + ─────── - ──── - 25⋅<x - 5>  + ──────── - 2⋅<x - 9> ⎟
+     ⎝               3       3                      3                 ⎠
     ────────────────────────────────────────────────────────────────────
-                                    E⋅I                                 
+                                    E⋅I
```

Comment:
This could cause the tests to fail, as it is copied from the interpreter output.

Reply:
I think the doctester ignores trailing whitespace, assuming that's what this is.